src/Animals/Ostrich.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Animals;
+
+class Ostrich extends Bird
+{
+    private $speed;
```

Comment:
I have seen the same code in SpeedTrait Why do you use copy-paste? Use trait one more time!